src/pages/settings/notifications/Notifications.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="flex flex-col p-4 bg-backgroundSecondary rounded-lg">
+    <h3 class="h3 mb-6 ">
+      Notifications you receive
+    </h3>
+    <div class="group" v-for="option of options">
+      <div class="flex items-center justify-between">
+        <p class="text-lg leading-[26px]">
```

Comment:
Let's move this font to a global class (regular-large).

I think it would be great to follow this approach with every font variation where we have comments in figma

<img width="286" alt="image" src="https://github.com/epicmaxco/vuestic-admin/assets/5394573/40d59942-3106-47d6-b8f9-f94b71f061ad">